emp-wrapper/src/main/java/org/ballerinalang/sf/ListenerUtil.java

```diff
@@ -75,8 +77,8 @@ public static Object startListener(String username, String password, ObjectValue
         connector = new EmpConnector(params);
```

Comment:
What is `EmpConnector ` we are creating here?

Reply:
EmpConnector is an instance of class EmpConnector in the listener module.